.ci/azure/windows.yml

```diff
@@ -240,7 +240,7 @@ jobs:
 
   - script: call $(SETUPVARS) && $(INSTALL_TEST_DIR)\paddle_tests --gtest_print_time=1 --gtest_output=xml:$(INSTALL_TEST_DIR)\TEST-Paddle.xml
     displayName: 'Paddle Frontend UT'
-    continueOnError: true
+    continueOnError: false
```

Comment:
Do we have to change this CI configuration?

Reply:
@ilya-lavrenov , do you have any comment?